frontend/app/src/shared/components/form/utils/getParentRelationship.ts

```diff
@@ -0,0 +1,13 @@
+import { genericSchemasAtom, nodeSchemasAtom } from "@/entities/schema/stores/schema.atom";
+import { store } from "@/shared/stores";
+
+export const getParentRelationship = (peer?: string) => {
```

Comment:
peer should be mandatory. If peer is undefined, it should be handled before calling this function.